src/components/Forms/ReplyForm/ReplyForm.js

```diff
@@ -0,0 +1,135 @@
+import cookie from 'react-cookies';
+import { useStyle } from './style';
+import config from '../../../config';
+import { serviceRequest } from '../../../services/serviceRequest';
+import { ErrorMessage } from '../../ErrorMessage';
+
+const path = '/api/v1/post/postreply';
+const domain = config.apiDomain;
+const preparePayload = (method, data) => {
+  const token = cookie.load('jwtToken');
+  const requestHeaders = { Authorization: token };
+  const url = `https://${domain}${path}`;
+  return {
+    method,
+    url,
+    data,
+    headers: requestHeaders,
+  };
+};
+
+export const ReplyForm = (props) => {
+  const classes = useStyle();
+
+  const [content, setContent] = useState('');
+  const [replyForm, setReplyForm] = useState(false);
+  const { commentID } = props;
+  const [isError, setIsError] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
+
+  const avatarURL = 'http://api.adorable.io/avatar/50/';
+
+  const handleChange = (event) => {
+    setContent(event.target.value);
+  };
+
+  const openReplyForm = () => {
+    setReplyForm(true);
+  };
+  const closeReplyForm = () => {
+    setContent('');
+    setReplyForm(false);
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const data = { commentID, content };
+      const payload = preparePayload('post', data);
+      const response = await serviceRequest(payload);
+      if (response.status && response.status === 'success') {
+        setContent('');
+        window.location.reload();
+      } else if (response.status && response.status === 'error') {
+        setIsError(true);
+        setErrorMsg(response.message);
+      } else {
+        throw new Error('Internal Service Error');
+      }
+    } catch (error) {
+      setErrorMsg('Internal Service Error');
+      setIsError(true);
+    }
+  };
+
+  return (
+    <Container maxWidth="lg">
+      <CssBaseline />
+      {isError && <ErrorMessage message={errorMsg} styles={{ color: 'red' }} />}
+      <Button
+        data-testid="showReplyButton"
+        className={classes.replyButton}
+        onClick={openReplyForm}
+      >
+        REPLY
+      </Button>
+      {replyForm && (
+        <form onSubmit={handleSubmit}>
+          <Grid container wrap="nowrap" className={classes.grid} spacing={1}>
+            <Grid item>
+              <Avatar
+                component="span"
+                alt="profile picture"
+                className={classes.avatar}
+                src={avatarURL + cookie.load('username')}
+              />
+            </Grid>
+            <Grid item xs>
+              <TextField
+                fullWidth
+                type="text"
+                multiline
+                autoComplete="on"
+                component="span"
+                data-testid="replyInput"
+                className={classes.input}
+                value={content}
+                onChange={handleChange}
+                inputProps={{ maxLength: 500 }}
```

Comment:
It might be helpful to add ` helperText` to let user know the number of the character limit. See PR #82 as a reference.